book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals # this is supposed to be array that houses the rentals made by persons
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    # rentals = []
```

Comment:
- [ ] The instance variable `@rentals` is not defined in the initialize method. Please define it so that you are able to use it in the Getter method at line 3.

Reply:
I wonder why I commented it out. Thanks